pennylane/__init__.py

```diff
@@ -33,6 +33,7 @@
 import pennylane.templates
 import pennylane.hf
 import pennylane.qchem
+from pennylane.hf import taper, symmetry_generators, paulix_ops
```

Comment:
This is done to make these 3 general user-facing functions available with `qml.name`. This guarantees that the tapering UI will not change when we bring these functions dipper inside pennylane (yet TBD where).

Reply:
deeper :wink: